### PR TITLE
Changes in prep for layouts release

### PR DESCRIPTION
**Pull request by Fercas123**
Scope of components to release: BorderLayout, FlexLayout, FlowLayout, StackLayout.

Changes in this PR:
- Removed separators from `SplitLayout`.
- Fix types for `FlowLayout`
- Remove examples that used Lab components. Removed from docs: `FlexLayout`, `FlowLayout`, `StackLayout`.
- Move examples that are using labs and are part of out of scope components out of in scope components.
- Change name for border items.

### CODE REVIEW

**Review by sion-docs — APPROVED** [review]

File: .changeset/beige-ravens-clap.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@jpmorganchase/uitk-core": minor
+---
+
+Fix `FlowLayout` types and remove separators prop from `SplitLayout`
```
Comment (Fercas123) [inline .changeset/beige-ravens-clap.md:5]:
```suggestion
- Fix `FlowLayout` types
- Remove separators prop from `SplitLayout`
- Rename BorderItem positions to follow border layout conventions
```

**Review by joshwooding — CHANGES_REQUESTED** [review]
BorderLayout docs still need to be updated with the renamed positions

File: packages/core/stories/layout/border-layout.stories.tsx [hunk 2]
```diff
@@ -181,166 +167,46 @@ BorderLayoutNoHeader.args = {};
...
+      <BorderItem position="west">
         <div
-          className={cx({
-            "border-layout-custom-form": checked,
-          })}
+          className="border-item border-left"
+          style={{ width: 100, height: 200 }}
         >
-          <FlowLayoutComposite />
+          <p>Left</p>
```
Comment (joshwooding) [inline packages/core/stories/layout/border-layout.stories.tsx:180]:
```suggestion
          <p>West</p>
```

File: packages/core/stories/layout/border-layout.stories.tsx [hunk 3]
```diff
@@ -181,166 +167,46 @@ BorderLayoutNoHeader.args = {};
...
-        <Header />
+      <BorderItem position="center">
+        <div
+          className="border-item border-main"
+          style={{
+            minWidth: 100,
+            height: "100%",
+          }}
+        >
+          <p>Main section</p>
```
Comment (joshwooding) [inline packages/core/stories/layout/border-layout.stories.tsx:191]:
```suggestion
          <p>Center</p>
```

File: packages/core/stories/layout/border-layout.stories.tsx [hunk 4]
```diff
@@ -181,166 +167,46 @@ BorderLayoutNoHeader.args = {};
...
-      <BorderItem position="main" as="main">
-        <div className="border-layout-contacts">
-          <h2>My contacts</h2>
-          <FlexLayoutComposite wrap />
+      <BorderItem position="east">
+        <div
+          className="border-item border-right"
+          style={{ width: 100, height: 200 }}
+        >
+          <p>Right</p>
```
Comment (joshwooding) [inline packages/core/stories/layout/border-layout.stories.tsx:199]:
```suggestion
          <p>East</p>
```

File: packages/core/stories/layout/border-layout.stories.tsx [hunk 5]
```diff
@@ -181,166 +167,46 @@ BorderLayoutNoHeader.args = {};
...
-
-      <BorderItem
-        position="bottom"
-        as="footer"
-        className="border-layout-contacts-footer"
-      >
-        <p>Contact lists get updated monthly.</p>
+      <BorderItem position="south">
+        <div className="border-item border-bottom" style={{ height: 50 }}>
+          <p>Bottom</p>
```
Comment (joshwooding) [inline packages/core/stories/layout/border-layout.stories.tsx:204]:
```suggestion
          <p>South</p>
```

File: packages/core/stories/layout/border-layout.stories.tsx [hunk 6]
```diff
@@ -181,166 +167,46 @@ BorderLayoutNoHeader.args = {};
...
-        <div className="border-layout-form-steps">
-          <SplitLayout
-            leftSplitItem={<FlowLayoutSimpleUsage />}
-            rightSplitItem={
-              <Pill label="In progress" className="border-layout-pill" />
-            }
-          />
+      <BorderItem position="north">
+        <div className="border-item border-header" style={{ height: 50 }}>
+          <p>Top</p>
```
Comment (joshwooding) [inline packages/core/stories/layout/border-layout.stories.tsx:172]:
```suggestion
          <p>North</p>
```

File: packages/core/stories/layout/border-layout.stories.tsx [hunk 7]
```diff
@@ -159,17 +132,30 @@ BorderLayoutNoLeftPanel.args = {};
...
+      <BorderItem position="east">
+        <div className="border-item border-right">
+          <p>Right</p>
+        </div>
       </BorderItem>
-      <BorderItem position="bottom">
-        <BottomItem />
+      <BorderItem position="south">
+        <div className="border-item border-bottom">
+          <p>Bottom</p>
```
Comment (joshwooding) [inline packages/core/stories/layout/border-layout.stories.tsx:157]:
```suggestion
          <p>South</p>
```

File: packages/core/stories/layout/border-layout.stories.tsx [hunk 8]
```diff
@@ -159,17 +132,30 @@ BorderLayoutNoLeftPanel.args = {};
...
+          }}
+        >
+          <p>Main section</p>
+        </div>
       </BorderItem>
-      <BorderItem position="right">
-        <RightItem />
+      <BorderItem position="east">
+        <div className="border-item border-right">
+          <p>Right</p>
```
Comment (joshwooding) [inline packages/core/stories/layout/border-layout.stories.tsx:152]:
```suggestion
          <p>East</p>
```

File: packages/core/stories/layout/border-layout.stories.tsx [hunk 9]
```diff
@@ -159,17 +132,30 @@ BorderLayoutNoLeftPanel.args = {};
...
-      <BorderItem position="main">
-        <MainItem />
+      <BorderItem position="center">
+        <div
+          className="border-item border-main"
+          style={{
+            minWidth: 100,
+          }}
+        >
+          <p>Main section</p>
```
Comment (joshwooding) [inline packages/core/stories/layout/border-layout.stories.tsx:147]:
```suggestion
          <p>Center</p>
```

File: packages/core/stories/layout/border-layout.stories.tsx [hunk 10]
```diff
@@ -159,17 +132,30 @@ BorderLayoutNoLeftPanel.args = {};
 const NoHeader: ComponentStory<typeof BorderLayout> = (args) => {
   return (
     <BorderLayout {...args}>
-      <BorderItem position="left">
-        <LeftItem />
+      <BorderItem position="west">
+        <div className="border-item border-left">
+          <p>Left</p>
```
Comment (joshwooding) [inline packages/core/stories/layout/border-layout.stories.tsx:137]:
```suggestion
          <p>West</p>
```

File: packages/core/stories/layout/border-layout.stories.tsx [hunk 11]
```diff
@@ -137,17 +97,30 @@ BorderLayoutNoRightPanel.args = {};
...
+      <BorderItem position="east">
+        <div className="border-item border-right">
+          <p>Right</p>
+        </div>
       </BorderItem>
-      <BorderItem position="bottom">
-        <BottomItem />
+      <BorderItem position="south">
+        <div className="border-item border-bottom">
+          <p>Bottom</p>
```
Comment (joshwooding) [inline packages/core/stories/layout/border-layout.stories.tsx:122]:
```suggestion
          <p>South</p>
```

File: packages/core/stories/layout/border-layout.stories.tsx [hunk 12]
```diff
@@ -137,17 +97,30 @@ BorderLayoutNoRightPanel.args = {};
...
+          }}
+        >
+          <p>Main section</p>
+        </div>
       </BorderItem>
-      <BorderItem position="right">
-        <RightItem />
+      <BorderItem position="east">
+        <div className="border-item border-right">
+          <p>Right</p>
```
Comment (joshwooding) [inline packages/core/stories/layout/border-layout.stories.tsx:117]:
```suggestion
          <p>East</p>
```

File: packages/core/stories/layout/border-layout.stories.tsx [hunk 13]
```diff
@@ -137,17 +97,30 @@ BorderLayoutNoRightPanel.args = {};
 const NoLeftPanel: ComponentStory<typeof BorderLayout> = (args) => {
   return (
     <BorderLayout {...args}>
-      <BorderItem position="header">
-        <HeaderItem />
+      <BorderItem position="north">
+        <div className="border-item border-header">
+          <p>Top</p>
```
Comment (joshwooding) [inline packages/core/stories/layout/border-layout.stories.tsx:102]:
```suggestion
          <p>North</p>
```

File: packages/core/stories/layout/border-layout.stories.tsx [hunk 14]
```diff
@@ -115,17 +62,30 @@ BorderLayoutAllPanels.args = {};
...
+          }}
+        >
+          <p>Main section</p>
+        </div>
       </BorderItem>
-      <BorderItem position="bottom">
-        <BottomItem />
+      <BorderItem position="south">
+        <div className="border-item border-bottom">
+          <p>Bottom</p>
```
Comment (joshwooding) [inline packages/core/stories/layout/border-layout.stories.tsx:87]:
```suggestion
          <p>South</p>
```

File: packages/core/stories/layout/border-layout.stories.tsx [hunk 15]
```diff
@@ -115,17 +62,30 @@ BorderLayoutAllPanels.args = {};
...
+      <BorderItem position="north">
+        <div className="border-item border-header">
+          <p>Top</p>
+        </div>
       </BorderItem>
-      <BorderItem position="left">
-        <LeftItem />
+      <BorderItem position="west">
+        <div className="border-item border-left">
+          <p>Left</p>
```
Comment (joshwooding) [inline packages/core/stories/layout/border-layout.stories.tsx:72]:
```suggestion
          <p>West</p>
```

File: packages/core/stories/layout/border-layout.stories.tsx [hunk 16]
```diff
@@ -115,17 +62,30 @@ BorderLayoutAllPanels.args = {};
 const NoRightPanel: ComponentStory<typeof BorderLayout> = (args) => {
   return (
     <BorderLayout {...args}>
-      <BorderItem position="header">
-        <HeaderItem />
+      <BorderItem position="north">
+        <div className="border-item border-header">
+          <p>Top</p>
```
Comment (joshwooding) [inline packages/core/stories/layout/border-layout.stories.tsx:67]:
```suggestion
          <p>North</p>
```

File: packages/core/stories/layout/border-layout.stories.tsx [hunk 17]
```diff
@@ -48,62 +19,38 @@ export default {
...
+      <BorderItem position="east">
+        <div className="border-item border-right">
+          <p>Right</p>
+        </div>
       </BorderItem>
-      <BorderItem position="bottom">
-        <BottomItem />
+      <BorderItem position="south">
+        <div className="border-item border-bottom">
+          <p>Bottom</p>
```
Comment (joshwooding) [inline packages/core/stories/layout/border-layout.stories.tsx:52]:
```suggestion
          <p>South</p>
```

File: packages/core/stories/layout/border-layout.stories.tsx [hunk 18]
```diff
@@ -48,62 +19,38 @@ export default {
...
+          }}
+        >
+          <p>Main section</p>
+        </div>
       </BorderItem>
-      <BorderItem position="right">
-        <RightItem />
+      <BorderItem position="east">
+        <div className="border-item border-right">
+          <p>Right</p>
```
Comment (joshwooding) [inline packages/core/stories/layout/border-layout.stories.tsx:47]:
```suggestion
          <p>East</p>
```

File: packages/core/stories/layout/border-layout.stories.tsx [hunk 19]
```diff
@@ -48,62 +19,38 @@ export default {
...
+      <BorderItem position="north">
+        <div className="border-item border-header">
+          <p>Top</p>
+        </div>
       </BorderItem>
-      <BorderItem position="left">
-        <LeftItem />
+      <BorderItem position="west">
+        <div className="border-item border-left">
+          <p>Left</p>
```
Comment (joshwooding) [inline packages/core/stories/layout/border-layout.stories.tsx:32]:
```suggestion
          <p>West</p>
```

File: packages/core/stories/layout/border-layout.stories.tsx [hunk 20]
```diff
@@ -48,62 +19,38 @@ export default {
...
-);
-
 const Template: ComponentStory<typeof BorderLayout> = (args) => {
   return (
     <BorderLayout {...args}>
-      <BorderItem position="header">
-        <HeaderItem />
+      <BorderItem position="north">
+        <div className="border-item border-header">
+          <p>Top</p>
```
Comment (joshwooding) [inline packages/core/stories/layout/border-layout.stories.tsx:27]:
```suggestion
          <p>North</p>
```

File: packages/core/stories/layout/border-item.stories.tsx [hunk 21]
```diff
@@ -17,73 +17,31 @@ export default {
...
-);
-
 const Template: ComponentStory<typeof BorderItem> = (args) => {
   return (
-    <BorderLayout columnGap={1} rowGap={1}>
-      <BorderItem position="header">
-        <HeaderItem />
+    <BorderLayout columnGap={1} rowGap={1} className="layout-container">
+      <BorderItem position="north" className="border-item">
+        <p>Header</p>
```
Comment (joshwooding) [inline packages/core/stories/layout/border-item.stories.tsx:24]:
```suggestion
        <p>North</p>
```

File: packages/core/stories/layout/border-item.stories.tsx [hunk 22]
```diff
@@ -17,73 +17,31 @@ export default {
...
+        <p>{args.position}</p>
       </BorderItem>
-      <BorderItem position="main">
-        <MainItem />
+      <BorderItem
+        position="center"
+        className="border-item layout-content"
+        style={{ minWidth: 100 }}
+      >
+        <p>Main</p>
```
Comment (joshwooding) [inline packages/core/stories/layout/border-item.stories.tsx:34]:
```suggestion
        <p>Center</p>
```

File: packages/core/stories/layout/border-item.stories.tsx [hunk 23]
```diff
@@ -17,73 +17,31 @@ export default {
...
       </BorderItem>
-      <BorderItem position="right">
-        <RightItem />
+      <BorderItem position="east" className="border-item">
+        <p>Right</p>
```
Comment (joshwooding) [inline packages/core/stories/layout/border-item.stories.tsx:37]:
```suggestion
        <p>East</p>
```

File: packages/core/stories/layout/border-item.stories.tsx [hunk 24]
```diff
@@ -17,73 +17,31 @@ export default {
...
       </BorderItem>
-      <BorderItem position="bottom">
-        <BottomItem />
+      <BorderItem position="south" className="border-item">
+        <p>Bottom</p>
```
Comment (joshwooding) [inline packages/core/stories/layout/border-item.stories.tsx:40]:
```suggestion
        <p>South</p>
```

**Review by joshwooding — APPROVED** [review]

**Pull request merged**